packages/components/src/badge/tests/chromatic/BadgeDot.chroma.jsx

```diff
@@ -0,0 +1,34 @@
+import { Badge } from "@components/badge";
+import { CircleBadge, IconBadge, SquareBadge, TextBadge } from "./BadgeUtils";
+import { Inline } from "@components/layout";
+
+export default {
+    component: Badge,
+    title: "Chromatic/Badge/dot"
+};
+
+
+export const Default = {
+    render: () => <SquareBadge variant="dot" />
+};
+
+export const Overlap = {
+    render: () => <Inline gap={8}>
+        <SquareBadge variant="dot" />
+        <CircleBadge variant="dot" />
+        <IconBadge variant="dot" />
+        <TextBadge variant="dot" />
+    </Inline>
+};
+
+export const Styling = {
+    render: () => <Inline>
+        <SquareBadge variant="dot" border="warning-7" />
+        <SquareBadge variant="dot" className="border-red" />
+        <SquareBadge variant="dot" style={{ border: "1px solid red" }} />
+    </Inline>
+};
+
+Default.storyName = "default";
+Overlap.storyName = "overlap";
```

Comment:
why are those storyName stacked at the bottom instead of directly after the elements. All the files before this one write the storyName right after the component

Reply:
an omission.